src/database/migrations/0073-alter-base_fields-add-file.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TYPE updated_field_type AS ENUM (
```

Comment:
Now I know that your "time to go multi-line" limit is either seven items or 80 columns, @slifty (comparing the one-item-per-line approach here with the all-on-one-line approach in [0054-update-field_type-enum.sql](https://github.com/PhilanthropyDataCommons/service/blob/main/src/database/migrations/0054-update-field_type-enum.sql)).

Reply:
I meant "your" in a very particular way that encompasses that kind of outsourcing, don't worry.